src/com/zone5cloud/core/ClientConfig.java

```diff
@@ -0,0 +1,59 @@
+package com.zone5cloud.core;
+
+import com.zone5cloud.core.oauth.AuthToken;
+
+import java.net.URL;
+
+public class ClientConfig {
+    // The initial authorization token to be used in the Authorization header for all requests which require it.
+    // token may be null and can be set later via setToken(), or automatically set with a login request
+    AuthToken token;
+
+    // Authentication service API key, Your OAuth app clientId (optional for S-Digital systems)
+    String clientID;
+
+    // Authentication service API secret, Cognito only. Set to null for Gigya keys.Your OAuth app secret (not needed for S-Digital systems)
+    String clientSecret;
+
+    // Users email address
+    String userName;
+
+    URL zone5BaseUrl;
+
+    public AuthToken getToken() {
+        return token;
+    }
+
+    public AuthToken setToken(AuthToken token) {
+        this.token = token;
+        return this.token;
+    }
```

Comment:
was this supposed to return the previous value? 

If not being used this can probably be a `public void setToken(AuthToken token)` signature to match the style of the other setters in this class.